spec/features/welcome/index_spec.rb

```diff
@@ -49,8 +49,6 @@
       expect(page).to have_content("Login with Google to:")
       expect(page).to have_content("- Set up a garden -")
       expect(page).to have_content("- Track your sensor data -")
-      expect(page).to have_content("- Connect with your community -")
-      expect(page).to have_content("- Track your garden's carbon impact -")
```

Comment:
Sad we couldn't implement this, it was a great idea!

Reply:
Agreed but I have faith that we can someday!